CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ## Bugfixes
 
+- Fix bash completion on macOS. See #2066 (@joshpencheon)
```

Comment:
```suggestion
- Fix bash completion on bash 3.x and bash-completion 1.x. See #2066 (@joshpencheon)
```

"on macOS" is inaccurate, there's nothing macOS specific here. The unmodified completion code would work on macOS too if one installs a recent enough bash and bash-completion@2 from homebrew.